Source/Android/jni/MainAndroid.cpp

```diff
@@ -131,6 +131,21 @@ void Host_UpdateTitle(const std::string& title)
   __android_log_write(ANDROID_LOG_INFO, DOLPHIN_TAG, title.c_str());
 }
 
+void Host_UpdateDiscordClientID(const std::string& client_id)
+{
+}
+
+bool Host_UpdateDiscordPresenceRaw(const std::string& details, const std::string& state,
+                                   const std::string& largeImageKey,
```

Comment:
Our convention is `snake_case` rather than `camelCase` for variable/parameter names. Ditto for the other occurrences in other files